### PR TITLE
PPF-240 Take into account admin access

### PR DESCRIPTION
**Pull request by LucWollants**
### Added
- Take into account admin access 

---
Ticket: https://jira.uitdatabank.be/browse/PPF-240

### CODE REVIEW

File: tests/Domain/Auth/Controllers/AccessControllerTest.php [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+
+        $this->accessController = new AccessController(
+            $this->auth0,
+            $this->integrationRepository,
+            ['admin@publiq.be'],
+            $this->logger
+        );
+    }
+
+    public function testInvalidTokenFails(): void
```
Comment (simon-debruijn) [inline tests/Domain/Auth/Controllers/AccessControllerTest.php:56]: Wasn't the convention to write tests in snake_case?

File: tests/Domain/Auth/Controllers/AccessControllerTest.php [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+        $this->integrationRepository->expects($this->never())
+            ->method('getById');
+
+        $response = $this->accessController->handle(
+            'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiZW1haWwiOiJhZG1pbkBwdWJsaXEuYmUiLCJpYXQiOjE1MTYyMzkwMjJ9.Haq7nxdrHejchsa22_JFUevoYF_vpEFBjW9iSdN2CF4',
+            '9469af87-3455-4853-a156-dbbc24965a26'
+        );
+
+        $this->assertEquals(200, $response->getStatusCode());
+    }
```
Comment (simon-debruijn) [inline tests/Domain/Auth/Controllers/AccessControllerTest.php:123]: Weird formatting

Comment (LucWollants) [reply tests/Domain/Auth/Controllers/AccessControllerTest.php:123]: The token is very long, could it be that? This call has 2 parameters, the first is the token and the second is the integration ID.

Comment (simon-debruijn) [reply tests/Domain/Auth/Controllers/AccessControllerTest.php:123]: It looks better now 👍🏻 

Probably due to github

**Pull request merged**